### PR TITLE
fix: minor improvements in error code display in SimplePowerBIReportEmbed

### PR DESCRIPTION
**Pull request by csm-thu**
- allow status code without text (and vice-versa)
 - remove dash between status code and status text

### CODE REVIEW

**Review by esasova — APPROVED** [review]
LGTM :+1:

File: src/charts/SimplePowerBIReportEmbed/SimplePowerBIReportEmbed.js [hunk 1]
```diff
@@ -50,19 +50,12 @@ const useStyles = makeStyles((theme) => ({
 }));
 
 function getErrorCode(labels, reports) {
-  let errorCode = labels.errors.unknown;
-  if (reports?.error?.status && reports?.error?.statusText) {
-    errorCode = `${reports?.error?.status} - ${reports?.error?.statusText}`;
-  }
-  return errorCode;
+  if (!reports?.error?.status && !reports?.error?.statusText) return labels.errors.unknown;
+  return `${reports?.error?.status ?? ''} ${reports?.error?.statusText ?? ''}`;
```
Comment (nborde-CSM) [inline src/charts/SimplePowerBIReportEmbed/SimplePowerBIReportEmbed.js:54]: This string syntax works without doubt but I am not very fan of it.

**Review by nborde-CSM — APPROVED** [review]
One observation but LGTM :+1:

File: src/charts/SimplePowerBIReportEmbed/SimplePowerBIReportEmbed.js (hunk 1 above)
Comment (nborde-CSM) [reply src/charts/SimplePowerBIReportEmbed/SimplePowerBIReportEmbed.js:54]: I think this work :
`return (reports?.error?.status ?? '') + ' '  + (reports?.error?.statusText ?? ''};`

In any case, your first solution works anyway.

**Pull request merged**